### PR TITLE
remove hardcode native token in token search

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- Backend return native token (address  '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
) from API for some use-cases.

- FE will remove hardcode native token in token search modal.

### CODE REVIEW

File: src/hooks/bridge/index.ts [hunk 1]
```diff
@@ -92,6 +87,7 @@ export const useTokensBalanceOfAnotherChain = (
       setBalances(balances as TokenAmountLoading[])
       return
     } catch (error) {
+      console.error('get balance chain err', chainId, error)
```
Comment (namgold) [inline src/hooks/bridge/index.ts:90]:
```suggestion
      console.error('get balance chain err', { chainId, error })
```

File: src/constants/index.ts [hunk 2]
```diff
@@ -142,7 +142,7 @@ export const COINGECKO_API_URL = 'https://api.coingecko.com/api/v3'
 export const KNC_COINGECKO_ID = 'kyber-network-crystal'
 
 export const ETHER_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
-export const ETHER_ADDRESS_SOLANA = 'eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
+export const ETHER_ADDRESS_SOLANA = 'So11111111111111111111111111111111111111111'
```
Comment (namgold) [inline src/constants/index.ts:145]: where did you get this?

Comment (nguyenhoaidanh) [reply src/constants/index.ts:145]: Backend sees it as a native token.
curently, FE only use ETHER_ADDRESS_SOLANA for price alert. 
price alert is ok with So11111111111111111111111111111111111111111 is present native.  
=> we will use this address for native sol

Comment (namgold) [reply src/constants/index.ts:145]: Can you give an example of API result? I dont get it where this address came from

Comment (nguyenhoaidanh) [reply src/constants/index.ts:145]: https://ks-setting.dev.kyberengineering.io/api/v1/tokens?query=sol&chainIds=101&page=1&pageSize=20
image

Comment (namgold) [reply src/constants/index.ts:145]: It's not on prod yet?

Comment (nguyenhoaidanh) [reply src/constants/index.ts:145]: yep, will add to admin when this pr release

**Review by namgold — APPROVED** [review]

**Pull request merged**